test/parallel/test-trace-events-environment.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+if (!common.isMainThread)
+  common.skip('process.chdir is not available in Workers');
+
+const names = [
+  'Environment',
+  'RunAndClearNativeImmediates',
+  'CheckImmediate',
+  'RunTimers',
+  'BeforeExit',
+  'RunCleanup',
+  'AtExit'
+];
+
+if (process.argv[2] === 'child') {
+  1 + 1;
```

Comment:
I'm assuming this is so the trace will have some content? If So could you add a comment?

Reply:
Done